src/jestTransformer.test.ts

```diff
@@ -22,10 +22,12 @@ describe('graphql-let/jestTransformer', () => {
       .filter(Boolean);
 
     const fullPath = join(cwd, fileName);
+    // @ts-expect-error
```

Comment:
I think we can use type casting and avoid using `@ts-expect-error`. What do you think? (Not sure if the blow suggestion can be applied)
```suggestion
Index: src/jestTransformer.test.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/jestTransformer.test.ts b/src/jestTransformer.test.ts
--- a/src/jestTransformer.test.ts	(revision 77da386b4f35d19bdefb242c5bd9d599ee8aa4f0)
+++ b/src/jestTransformer.test.ts	(revision 7d1a588958221a3d4cce57fad8af0c2608ea8577)
@@ -1,16 +1,17 @@
+import { TransformOptions } from '@jest/transform';
 import { Config } from '@jest/types';
 import { join } from 'path';
-import jestTransformer from './jestTransformer';
+import jestTransformer, { JestTransformerOptions } from './jestTransformer';
 import compiler from './lib/__tools/compile';
 import { prepareFixtures } from './lib/__tools/file';
 
 let cwd: string;
-let config: Partial<Config.ProjectConfig>;
+let config: Config.ProjectConfig;
 
 describe('graphql-let/jestTransformer', () => {
   beforeAll(async () => {
     [cwd] = await prepareFixtures(__dirname, '__fixtures/jestTransformer');
-    config = { rootDir: cwd };
+    config = { rootDir: cwd } as Config.ProjectConfig;
   });
 
   test('transforms .graphql', async () => {
@@ -22,12 +23,10 @@
       .filter(Boolean);
 
     const fullPath = join(cwd, fileName);
-    // @ts-expect-error
     const { code: transformedContent } = jestTransformer.process(
       fileData!,
       fullPath,
-      // @ts-expect-error
-      { config },
+      { config } as TransformOptions<JestTransformerOptions>,
     ) as { code: string };
     expect(removeSourcemapReference(transformedContent)).toMatchSnapshot();
   });
Index: src/jestTransformer.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/jestTransformer.ts b/src/jestTransformer.ts
--- a/src/jestTransformer.ts	(revision 77da386b4f35d19bdefb242c5bd9d599ee8aa4f0)
+++ b/src/jestTransformer.ts	(revision 7d1a588958221a3d4cce57fad8af0c2608ea8577)
@@ -1,4 +1,4 @@
-import { Transformer } from '@jest/transform';
+import { SyncTransformer } from '@jest/transform';
 import { ProjectConfig } from '@jest/types/build/Config';
 import { readFileSync } from 'fs';
 import { relative } from 'path';
@@ -7,7 +7,7 @@
 import { createHash } from './lib/hash';
 import { createPaths } from './lib/paths';
 
-type JestTransformerOptions = {
+export type JestTransformerOptions = {
   configFile?: string;
   subsequentTransformer?: string;
 };
@@ -27,7 +27,7 @@
   return {};
 }
 
-const jestTransformer: Transformer<JestTransformerOptions> = {
+const jestTransformer: SyncTransformer<JestTransformerOptions> = {
   getCacheKey(sourceText, sourcePath, options) {
     const configString = options?.configString || options;
```

Reply:
Tell me what you think about my quick try: https://github.com/piglovesyou/graphql-let/commit/7d1a588958221a3d4cce57fad8af0c2608ea8577

Reply:
I'm not a big fan of casting at general, although quick try LGTM.

Reply:
Thank you for sharing your thought. Yes, using type casting to make mock data for testing purposes won't bite you. Would you mind including the change in this PR?

Reply:
No worry, I'll add the commit. Thank you very much!!